networkx/algorithms/approximation/dominating_set.py

```diff
@@ -41,6 +41,15 @@ def min_weighted_dominating_set(G, weight=None):
         each node in the graph and `w(V^*)` denotes the sum of the
         weights of each node in the minimum weight dominating set.
 
+    Examples
+    --------
+    >>> from networkx.algorithms.approximation.dominating_set import min_weighted_dominating_set
+    >>> G = nx.Graph()
+    >>> edges = [(0, 1),(0, 3),(1, 2),(1, 3),(1, 4)]
+    >>> G.add_edges_from(edges)
```

Comment:
This is more concisely written as:

```suggestion
    >>> G = nx.Graph([(0, 1), (0, 3), (1, 2), (1, 3), (1, 4)])
```

Also note the spaces between the tuples - we use `black` formatting for the docstring examples as well. This should be automatically applied when you commit if you have the `pre-commit` hooks installed: take a look at the [contributor guide](https://networkx.org/documentation/latest/developer/contribute.html#development-workflow) for info on how to properly set this up.